shell/platform/fuchsia/flutter/runner.cc

```diff
@@ -46,6 +51,28 @@ uintptr_t GetICUData(const fuchsia::mem::Buffer& icu_data) {
   return 0u;
 }
 
+// Initializes the timezone data if available.  Timezone data file in Fuchsia
+// is at a fixed directory path.  Returns true on success.  As a side effect
+// sets the value of the environment variable "ICU_TIMEZONE_FILES_DIR" to a
+// fixed value which is fuchsia-specific.
+bool InitializeTZData() {
+  // Try opening the path to check if present.  No need to verify that it is a
+  // directory since ICU loading will return an error if the TZ data path is
+  // wrong.
+  int fd = openat(AT_FDCWD, kICUTZDataDir, O_RDONLY);
+  if (fd < 0) {
+    return false;
+  }
+  // Should we overwrite this instead?
+  setenv("ICU_TIMEZONE_FILES_DIR", kICUTZDataDir, 0 /* Don't overwrite if
+													   exists */);
+  if (!close(fd)) {
+    FML_LOG(WARNING) << "Could not close: " << kICUTZDataDir;
```

Comment:
I think it will be useful to log `strerror(errno)` as well in this case, this might also be useful when `openat` fails.

Reply:
Done.